bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,327 @@
+    repository: https://github.com/vmware/nsx-container-plugin/operator
+    support: VMware
+  name: nsx-container-plugin-operator.v3.1.2
+spec:
+  apiservicedefinitions: {}
+  replaces:
+  - nsx-container-plugin-operator.v0.2.0
+  customresourcedefinitions:
+    owned:
+    - description: NcpInstall is the Schema for the ncpinstalls API
+      displayName: NcpInstall
+      kind: NcpInstall
+      name: ncpinstalls.operator.nsx.vmware.com
+      resources:
+      - kind: Deployment
+        name: A Kubernetes Deployment for the Operator
+        version: v1
+      - kind: Configmap
+        name: A Configmap with parameter for setting up Operands
+        version: v1
+      - kind: NcpInstall
+        name: this operator's CR
+        version: v1
+      - kind: ClusterOperator
+        name: nsx-ncp cluster operator
+        version: v1
+      - kind: Network
+        name: Openshift's cluster network
+        version: v1
+      - kind: Daemonset
+        name: A Daemonset
+        version: v1
+      - kind: Pod
+        name: A Pod
+        version: v1
+      - kind: Secret
+        name: It's a secret, I can't tell you
+        version: v1
+      - kind: Node
+        name: A Kubernetes Node
+        version: v1
+      - kind: Status
+        name: A Kubernetes resource status
+        version: v1
+      specdescriptors:
+      - description: the replica numbers for the nsx-ncp deployment
+        path: ncpReplicas
+      statusdescriptors:
+      - description: standard conditions field for Kubernetes resources
+        path: conditions
+      version: v1
+  description: An operator which provides NSX as default network for an Openshit cluster.
+    Simplifies the process of installing and upgrading the NSX Container plugin (NCP)
+    components running in the Openshift cluster. The operator also allows for detailed
+    monitoring of NCP components and reacts to configuration changes.
+  displayName: NSX Container Plugin Operator
+  install:
+    spec:
+      deployments:
+      - name: nsx-ncp-operator
+        template:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: nsx-ncp-operator
+          template:
+            metadata:
+              labels:
+                name: nsx-ncp-operator
+            spec:
+              containers:
+              - command:
+                - /bin/bash
+                - -c
+                - nsx-ncp-operator --zap-time-encoding=iso8601
+                env:
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: nsx-ncp-operator
+                - name: NCP_IMAGE
+                  value: nsx-ncp:latest
+                - name: WATCH_NAMESPACE
+                  value: nsx-system-operator
+                image: docker.io/vmware/nsx-container-plugin-operator:v3.1.2.1
```

Comment:
Just want to confirm the image tag is `v3.1.2.1`?

Reply:
Yes, we published a new release to ensure wait_for_security_policy_sync was disabled.

I might need to push another tag as image might need to be rebuilt to get latest version of package (to pass security checks).

In the meanwhile we can go with this tag.